sormas-backend/src/main/java/de/symeda/sormas/backend/feature/FeatureConfigurationFacadeEjb.java

```diff
@@ -254,6 +257,35 @@ public boolean isFeatureDisabled(FeatureType featureType) {
 		return em.createQuery(cq).getSingleResult() > 0;
 	}
 
+	@Override
+	@SuppressWarnings("unchecked")
+	public boolean isPropertyValue(FeatureType featureType, FeatureTypeProperty property, boolean expectedPropertyValue) {
```

Comment:
1. I would name this method `isPropertyEnabled`
2. Missing unit test
3. Also define `getPropertyValue:String` or `getPropertyValue(..., Class<P> propertyClass):P` or wait for that use cases (4. bullet point of the ticket)? That would be needed in my opinion to completly deliver the feature (not only for `boolean`).

Reply:
See above. `isPropertyEnabled` would not be an appropriate name for what the method is supposed to do. I'll add Javadoc. `getPropertyValue` is no longer needed because I replaced it with the `isPropertyValue` method which, in my opinion, is more valuable for this use case right here. We can add `getPropertyValue` later when we need it, but right now it would be dead code.

Unit test has been added, but it only covers the default values. Adding explicit JSON values to the database leads to a data conversion error because H2 has issues with the JSON converting that we use for PostgreSQL. This should be tackled separately from this issue.

Reply:
Not being able to configure this in a unit test might result in other features not properly tested then. But it seems there is a solution: Update _hibernate-types-52_ (see developer comment here: https://stackoverflow.com/questions/39620317/how-to-map-a-json-column-with-h2-jpa-and-hibernate/67467069#67467069)